features/nanostack/FEATURE_NANOSTACK/mbed-mesh-api/source/thread_tasklet.c

```diff
@@ -204,6 +225,29 @@ void thread_tasklet_parse_network_event(arm_event_s *event)
     }
 }
 
+void thread_tasklet_poll_network_status()
+{
+    /* Check if we do have an IP */
+    uint8_t temp_ipv6[16];
+    if (arm_net_address_get(thread_tasklet_data_ptr->nwk_if_id, ADDR_IPV6_GP, temp_ipv6) == 0) {
+        /* Check if this is the same IP than previously */
+        if (memcmp(temp_ipv6, thread_tasklet_data_ptr->ip, 16) == 0) {
+            return;
+        } else {
+            memcpy(thread_tasklet_data_ptr->ip, temp_ipv6, 16);
+            if (memcmp(thread_tasklet_data_ptr->ip, thread_tasklet_data_ptr->link_config.mesh_local_ula_prefix, 8) == 0) {
```

Comment:
Do you need to re-read the mesh_local_ula prefix too? Is that only set on the initial connect, or is it being updated?

Reply:
Yes, correct, that isn't updated, so need to read and update from the thread_management_configuration. I'll make a commit, I tested it already.